src/main/webapp/WEB-INF/ResetStatistics.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page import="java.sql.*,java.util.Random,java.io.*"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Reset Statistics</title>
+</head>
+<body>
+	<%
+out.println("You can view REset Statistics here!");
```

Comment:
Fix typo here please (REset)

Reply:
i fixed the typo Chris! check it out and tell me.